wordpress-mobile/WordPress-iOS-Shared
# Support Swift 5

Closes #200.

This PR:
1. Updates pods to support Swift 5
2. Make Xcode recommended changes to support Swift 5
3. Updates the project file to remove a warning.

Changes made by Platform 9 3/4: updated tooling to support Xcode 10.2 and Swift 5 🌟 

First reviewer wins! 🏆 

## To test
- `bundle exec pod install` ensure no warnings are spit out in the command line
- build the framework
- run the unit tests
- no warnings should appear in Xcode

## To test in a host app
- check out this PR for WCiOS as the host app: https://github.com/woocommerce/woocommerce-ios/pull/924
- `bundle exec pod install`. No warnings should appear in the command line.
- run unit tests
- run app
